packages/remix/src/utils/instrumentServer.ts

```diff
@@ -206,6 +222,34 @@ function makeWrappedLoader(origAction: DataFunction): DataFunction {
   return makeWrappedDataFunction(origAction, 'loader');
 }
 
+function makeWrappedMeta(origMeta: MetaFunction | HtmlMetaDescriptor = {}): MetaFunction {
+  return function (
+    this: unknown,
+    args: { data: AppData; parentsData: RouteData; params: Params; location: Location },
+  ): HtmlMetaDescriptor {
+    let origMetaResult;
+    if (origMeta instanceof Function) {
+      origMetaResult = origMeta.call(this, args);
+    } else {
+      origMetaResult = origMeta;
+    }
+
+    const scope = getCurrentHub().getScope();
+    if (scope) {
+      const span = scope.getSpan();
+
+      if (span) {
+        return {
+          ...origMetaResult,
+          'sentry-trace': span.toTraceparent(),
```

Comment:
RE: `baggage` meta tag

Don't take this suggestion literally (e.g. still needs imports and all) but I think this is how it could work.

Slightly above, get the active `transaction` (from the scope) and then here:
```suggestion
          'sentry-trace': span.toTraceparent(),
          baggage: serializeBaggage(transaction.getBaggage()),
```